src/main/java/com/cqse/teamscaleupload/TeamscaleUpload.java

```diff
@@ -70,7 +70,7 @@ private Input(Namespace namespace) {
             this.url = HttpUrl.parse(namespace.getString("server"));
             this.message = namespace.getString("message");
             this.keystorePathAndPassword = namespace.getString("trusted_keystore");
-            this.validateSsl = namespace.getBoolean("validate_ssl") || keystorePathAndPassword != null;
+            this.validateSsl = !namespace.getBoolean("insecure") || keystorePathAndPassword != null;
```

Comment:
I'd expect that if I specify --insecure it will disable SSL verify, even if --trusted_keystore is given. Other option, raise error if both are set. WDYT?

Reply:
good point! warning now